### DISCUSSION TITLE
After purchasing an item using stripe payment, it shows successful in stripe but in the orders it still stays in "adding items" state. How to fix it?

### DISCUSSION THREAD

**Discussion by AtharTasdiq** [vendurehq/vendure#3126] [Q&A]
Here is the scenario 1,
- Suppose there are two separate channels(Channel 1 and Channel 2) and a default channel.
- There is only Stripe Payment method for all the channels (Assigned to all channels).
- A new product is uploaded in default channel.
- New product is assigned to Channel 1
- Tried to buy the new product from Channel 1's "Vendure Remix Storefront".
- It shows order is received but still stays on the cart in "Vendure Remix Storefront". In stripe transaction it is also "Successful".
- In the orders in "Vendure Dashboard" the order state is in "Adding items".

Here is the scenario 2,
- Same as before but this time, a new product uploaded in default channel but assigned to both "Channel 1" and "Channel 2".
- Tried to buy the new product from Channel 1's "Vendure Remix Storefront".
- Now everything is working fine (Order is received, Cart is empty, Stripe is successful, Order state is "Payment settled")

#### Now my question is how can I fix this issue? I need to upload products in default channel and assign them to different channels as I want. I do not want to assign every product to each channels. I need to fix it as soon as possible.

**Comment by twlite** [comment]
I'm experiencing this issue too, did you find a solution @AtharTasdiq?

**Reply by AtharTasdiq** [reply]
Bro if you have installed Vendure Multivendor Plugin then uninstall that, this issue is created by the multivendor plugin. When I commented the Multivendor code from Vendure Config file then it solved the issue, then I uninstalled the Plugin and now it is working fine.
